manifests/lcm_config.pp

```diff
@@ -0,0 +1,12 @@
+define dsc::lcm_config (
+  $refresh_mode = 'Disabled'
+) {
+
+  validate_re($refresh_mode, '^(Disabled|Pull|Push)$', 'refresh_mode must be one of \'Disabled\', \'Push\', \'Pull\'')
```

Comment:
We haven't decided on that one yet. It depends on whether you can set the mode case insensitively or not.